webclient/.bowerrc

```diff
@@ -0,0 +1,4 @@
+{
+  "directory": "bower_components",
```

Comment:
I think the directory is named bower_components by default in order to prevent confusions between bower dependencies and Ember components. But I can rename it components if you'd prefer.